ide/che-core-ide-api/src/main/java/org/eclipse/che/ide/api/vcs/HasVcsStatus.java

```diff
@@ -0,0 +1,32 @@
+/*******************************************************************************
+ * Copyright (c) 2012-2017 Codenvy, S.A.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   Codenvy, S.A. - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.che.ide.api.vcs;
+
+/**
+ * Indicates that specified resource has VCS status attribute.
```

Comment:
Can you please elaborate on what it means to have VCS status _attribute_?
Does it mean that each file has VCS status if the project is under VCS?